src/app/[roomId]/page.tsx

```diff
@@ -168,19 +125,16 @@ export default function ChatRoomPage() {
 
   // Set alias for presence tracking when user joins
   useEffect(() => {
-    if (alias && isConnected && socketRef.current) {
-      socketRef.current.emit("set-alias", alias);
+    if (alias && isConnected) {
+      setSocketAlias(alias);
     }
-  }, [alias, isConnected]);
+  }, [alias, isConnected, setSocketAlias]);
 
   if (error) {
     return (
       <div className="min-h-screen flex flex-col items-center justify-center">
-        <div className="flex items-center gap-4 mb-10">
-          <h1 className="text-5xl font-bold font-mono bg-yapli-teal bg-clip-text text-transparent pb-2">
-            yapli
-          </h1>
-          <Logo size={32} className="mt-2" />
+        <div className="mb-10">
+          <Brand />
         </div>
```

Comment:
```suggestion
        <Brand className="mb-10" />
```
I think the div is not required here